ledger-core/virtual/integration/outgoing_test.go

```diff
@@ -28,7 +29,21 @@ import (
 )
 
 // A.Foo calls B.Bar, pulse changes, outgoing has proper token
-func TestVirtual_CallMethodOutgoing_WithPulseChange(t *testing.T) {
+/*
+VCallRequest [A.Foo]
+ExecutionStart
+change pulse -> secondPulse
+VStateReport [A]
+VDelegatedCallRequest [A]
+VCallRequest [B.Bar] + first token
+change pulse -> third pulse
+NO VStateReport
+VDelegatedCallRequest [A] + first token
+VCallRequest [B.Bar] + second token
+VCallResult [A.Foo] + second token
+VDelegatedRequestFinished [A] + second token
+*/
+func TestVirtual_CallMethodOutgoing_WithTwicePulseChange(t *testing.T) {
 	t.Log("C5141")
```

Comment:
I'm gonna fix the case in testrail after merge

Reply:
why you can't done this now?

Reply:
now we have simple process - reviewer see code, logic and testrail case. i have question about this changes - test with one pulse change(testrail old case) is not need? why?

Reply:
because the case is already passed in testrail, I don't want to make a misunderstanding while the branch isn't merged.

Reply:
test with two pulse change contains one pulse change, so that test with one pulse change will duplicate this test.  
if the problem occurs in one pulse change we will have two fails, I do think that it's better to have one test.

Reply:
done